src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import Button from 'components/Button';
+import Card from 'components/Card';
+import Text from 'components/Text';
+import { URLS } from 'configs/urls';
+import { RecipeData } from 'types/RecipeData';
+import { ColorType } from 'types/common';
+import styles from './styles.module.scss';
+
+type CardListProps = {
+  cardsData: RecipeData[];
+};
+
+export const CardList: React.FC<CardListProps> = ({ cardsData }) => {
+  return (
+    <>
+      {cardsData.map((card) => (
+        <Card
+          title={card.title}
+          image={card.image}
+          subtitle={card?.nutrition?.ingredients.map((ingredient) => ingredient.name).join(' + ')}
+          captionSlot={card?.readyInMinutes ? card?.readyInMinutes + ' minutes' : 'unknown'}
+          actionSlot={
+            <Link to={`${URLS.recipe + '/' + card.id}`} style={{ textDecoration: 'none' }}>
```

Comment:
Лучше в URLS хранить не просто строку в данном случае, а функцию, которая будет принимать id и возвращать готовый урл
```suggestion
          actionSlot={
            <Link to={`${URLS.recipe.create(card.id)}`} style={{ textDecoration: 'none' }}>
```

Reply:
done